docs/testnet-deployments.md

```diff
@@ -551,6 +552,30 @@
 
 ## Okx Testnet
 
+| Contract           | Address                                    |
+| ------------------ | ------------------------------------------ |
+| BabelCore          | 0xed0ac2eea2cec5c27e509e29814574603587f682 |
```

Comment:
The addresses got messed up. Check it please. The Okx testnet and  moonbase testnet addresses got messed up it seems.

Reply:
These are the correct addresses  just verified it on the both Chains.